### PR TITLE
[EBE-4709] Color refactoring

### PR DESCRIPTION
**Pull request by pascualin**
This Pull Request includes Pull Request #112.

Deleted duplicate colors on variables.less and updated all the .less files to point to the right unique color.

### CODE REVIEW

**Comment by pascualin** [comment]
@mattnickles please review

**Comment by jsmecham** [comment]
The color values on the tokens page need to be updated.

File: src/less/accountSwitcher.less [hunk 1]
```diff
@@ -74,7 +74,7 @@ account-switcher {
     }
 
     .current {
-      color: @account-switcher-text;
+      color: @gray-200;
```
Comment (mattnickles) [inline src/less/accountSwitcher.less]: change to @account-switcher-text

File: src/less/accountSwitcher.less [hunk 2]
```diff
@@ -116,9 +117,9 @@ account-switcher {
         input {
           width: 100%;
           font-size: 16px;
-          background: @account-switcher-bg;
-          border-color: @account-switcher-border-color;
-          color: @account-switcher-input-text;
+          background: @switcher-bg;
+          border-color: @gray-400;
+          color: @gray-200;
```
Comment (mattnickles) [inline src/less/accountSwitcher.less]: change to @account-switcher-input-text

File: src/less/accountSwitcher.less [hunk 3]
```diff
@@ -169,27 +171,35 @@ account-switcher {
...
       .loading-list-view-items {
+        background: @switcher-bg;
         border-bottom-color: @account-switcher-border-color;
-        color: @gray-300;
+        color: @gray-200;
       }
 
       .list-view-item {
-        border-bottom-color: @account-switcher-border-color;
+        border-bottom-color: @black;
```
Comment (mattnickles) [inline src/less/accountSwitcher.less]: change to @account-switcher-border-color

File: src/less/variables.less [hunk 4]
```diff
@@ -223,28 +227,28 @@
...
 //
-@account-switcher-header-bg:          @gray-700;
-@account-switcher-bg:                 @gray-800;
+@account-switcher-header-bg:          @black;
+@account-switcher-bg:                 @switcher-bg;
 @account-switcher-text:               @gray-200;
-@account-switcher-border-color:       @gray-700;
-@account-switcher-active-bg:          @gray-900;
+@account-switcher-border-color:       @black;
+@account-switcher-active-bg:          @black;
```
Comment (mattnickles) [inline src/less/variables.less]: Similar confusion here with account-switcher-active-bg and switcher-current-bg. Can we restore account-switcher-active-bg as the active background color, set it's value to #000 here and remove switcher-current-bg? Or add #000 to our color palette?

File: src/less/buttons.less [hunk 5]
```diff
@@ -3,7 +3,7 @@
   border: 1px solid transparent;
   border-radius: @border-radius-small;
   box-shadow: none;
-  color: #fff;
+  color: @black;
```
Comment (mattnickles) [inline src/less/buttons.less]: Thoughts on changing this to @text-color? I don't want to comment on all of them but there's about 10 other places we could be leveraging that token.

**Comment by pascualin** [comment]
@jsmecham could you take a look at the new updates please?

File: package.json [hunk 6]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "cyclops",
-  "version": "1.7.1",
+  "version": "1.8.0",
```
Comment (jsmecham) [inline package.json]: Should probably rebase this on master, as this should be 1.8.1 now.

File: src/less/buttons.less (hunk 5 above)
Comment (jsmecham) [inline src/less/buttons.less]: I like that.

**Comment by jsmecham** [comment]
Looks like this has some inverse theme naming. This should be rebased on master since the inverse styles have been removed.

**Comment by pascualin** [comment]
@mattnickles please review

**Pull request merged**